depends/packages/qt.mk

```diff
@@ -1,22 +1,20 @@
 package=qt
-$(package)_version=5.15.11
+$(package)_version=5.15.13
 $(package)_download_path=https://download.qt.io/official_releases/qt/5.15/$($(package)_version)/submodules
 $(package)_suffix=everywhere-opensource-src-$($(package)_version).tar.xz
 $(package)_file_name=qtbase-$($(package)_suffix)
-$(package)_sha256_hash=425ad301acd91ca66c10c0dabee0704e2d0cd2801a6b670115800cbb95f84846
+$(package)_sha256_hash=4cca51dcc1f22ceeee6b3e33cd1c3a60b14e85e24644dca3af89a2c2989ab809
 $(package)_linux_dependencies=freetype fontconfig libxcb libxkbcommon libxcb_util libxcb_util_render libxcb_util_keysyms libxcb_util_image libxcb_util_wm
 $(package)_qt_libs=corelib network widgets gui plugins testlib
 $(package)_linguist_tools = lrelease lupdate lconvert
 $(package)_patches = qt.pro
 $(package)_patches += qttools_src.pro
 $(package)_patches += mac-qmake.conf
 $(package)_patches += fix_qt_pkgconfig.patch
-$(package)_patches += fix_qt_placeholders.patch
```

Comment:
this is extra from upstream commit?

Reply:
As mentioned in the PR description, `fix_qt_placeholders.patch` was a patch introduced by [dash#5596](https://github.com/dashpay/dash/pull/5596) and no longer necessary as the problem that was addressing has been resolved in the version of Qt that we're bumping to.